.github/workflows/python_publish.yml

```
@@ -32,8 +32,7 @@ jobs:
       - if: runner.os == 'Linux'
         run: |
           sudo apt-get update -q -q
-          sudo apt-get install --yes espeak-ng libespeak1 alsa-utils
-          sudo modprobe snd-dummy
+          sudo apt-get install --yes espeak-ng libespeak1 alsa-utils ffmpeg
```

Sort dependencies to make it easy to spot missing deps and nearly impossible to add duplicates.
```suggestion
          sudo apt-get install --yes alsa-utils ffmpeg espeak-ng libespeak1
```